### DISCUSSION TITLE
Why is storage filling up with small 2MB MP4 files ?

### DISCUSSION THREAD

**Discussion by MrSiO** [blakeblackshear/frigate#9221] [Ask A Question] [answered]
Hi all.

I noticed my disk space is getting filled by frigate constantly even without any events & recordings triggered.
Folders in **/media/frigate/recordings/2024-01-XX/0X/FRONT** were automatically created and filled with small, 2MB MP4 videos with what looks like to be the time as namefile.  Eg:  **01.39.MP4**

(see picture)

![Frigate-2MB-MP4-Files](https://github.com/blakeblackshear/frigate/assets/49870576/6d094d97-5cb7-4373-ae53-4ab772a6d7af)

Anyone knows if this is related to my recording configuration mode set to Motion?  ( I could not see any motion in the videos tough..)

```
record:
  enabled: True
  retain:
    days: 7
    mode: motion
```

Note: Those small MP4 arent seen/available trough the frigate UI nor the Home Assistant media...  

Thx guys.

**Comment by NickM-27** [comment, marked as answer]
They are available in the frigate ui, check the 24/7 recording reviewer. The docs for 0.13 have a guide on tuning motion detection

**Reply by MrSiO** [reply]
Hi.
I had changed the "active_object" to "motion" as part of the ticket's #9089 final resolution to fix my manual trigger issue.
I will set it back as it was and work on improving Object Detection to fix the manuel trigger issue I first had (#9089).


Thx for your support!

**Reply by NickM-27** [reply]
No need to set it back, just tune motion detection to be less sensitive
